README.md

```diff
@@ -576,7 +581,46 @@ capabilitiesToBeDropped:
   - SETFCAP #Set file capabilities.
 ```
 
-This can be overridden by using `-d` flag and providing your own defaults in the yaml format as shown above.
+This can be overridden by using `-k` flag and providing your own defaults in the yaml format as shown below.
```

Comment:
if we rename it to audit config and `-a` is free we could use that or just support a long flag which should be fine too.